### PR TITLE
Releasing version 2.4.40

### PR DESCRIPTION
**Pull request by dshelbyo**
## Added
- Support for sparse diskgroup option with Exadata shape in the following command:
   - (``oci db system launch``)
-  Support for Data Guard on VM DB Shape
 - Support create option with-new-db-system along with from-existing-db-system
   - (``oci db data-guard-association create with-new-db-system``)
 - Support for tagging Zones in the DNS service.
 - Block Storage paravirtualized-encryption-in-transit feature
   - Ability to enable encryption-in-transit for paravirtualized volume attachment for both boot volumes and data volumes (``oci compute volume-attachment attach-paravirtualized-volume``)
 - Support for resetting idp scim client as part of Identity Service.
   - (``oci iam scim-client-credentials reset-idp-scim-client --identity-provider-id``)
 - Support for updating user capabilities as part of Identity Service.
   - (``oci iam user update-user-capabilities --user-id``)
 
## Changed
- New Attribute ``is-latest-for-major-version`` is included in (``oci db version list``) response
- pyOpenSSL was upgraded to version 17.5.0 and cryptography to version 2.1.4 to address a vulnerability identified on GitHub as CVE-2018-1000808.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -106,10 +141,6 @@ Changed
 * New Attribute ``dbVersion`` is included in the GET Response for Autonomous Transaction Processing Database and Autonomous Data Warehouse.
 * New Attribute ``allConnectionStrings`` is included in the GET Response for Autonomous Transaction Processing Database and Autonomous Data Warehouse.
 
-Known Issues
```
Comment (viralmodi) [inline CHANGELOG.rst]: Instead of removing this, we should add a line to our latest changelog saying this has been fixed

Comment (mross22) [reply CHANGELOG.rst]: The rationale was that fixing this was a server side change so we wanted to remove it from 'Known Issues' so future users of this version won't think that it is a limitation of this specific CLI version.

File: tests/output/inline-help/iam.txt [hunk 2]
```diff
@@ -1323,6 +1325,45 @@ Options:
...
+  A group created in an identity provider that can be mapped to a group in OCI
+
+Options:
+  -?, -h, --help  Show this message and exit.
+
+Commands:
+  list  Lists the identity provider groups.
+
+++++++++++++++++++++++++++++++++++++++++++++++
+$ oci iam identity-provider-group list --help
```
Comment (viralmodi) [inline tests/output/inline-help/iam.txt]: This command seems to be missing in changelog

Comment (mross22) [reply tests/output/inline-help/iam.txt]: Good catch thanks! Added to changelog

**Pull request merged**